src/env.js

```diff
@@ -33,6 +33,7 @@ Sk.python2 = {
     absolute_import: null,
     unicode_literals: false,
     // skulpt specific
+    python3: false,
```

Comment:
so there are a few things that need their own flag because they are importable from `__future__` and when we only had a few python3 features it made sense from a production environment to be able to turn them on only for new stuff so you don't break old code out there. But since there are now probably more python2 versions you want to turn off instead of python3 version to turn on. We felt this would do for now and we see if it makes sense to switch every difference between py3 and py2.

Reply:
So moving forward, we can/should just use the python3 future flag for things we add instead of creating new future flags?

Reply:
Yes, unless there's a particular reason to make it switchable independently.

Reply:
Sounds good!